add more projects info sorted in the order of A-Z and a-z.

junedwx opened the pull request.

HuKeping commented:
CI failed because  http://git.netfilter.org/arptables does not support `git clone --depth 1`

```
Cloning into 'arptables'...
fatal: dumb http transport does not support shallow capabilities
```

We've add ` https://git.netfilter.org/arptables`  (https not http) into the unshallow_support list, see https://github.com/HuKeping/upstream/blob/master/validate_access.bash#L8

Use `https://git.netfilter.org/arptables` as the upstream of arptables would fix the failure.

DINGwenxin123 committed:
Add more projects info sorted in the order of A-Z and a-z.

modeify upstream.yaml add more projects info sorted in the order of A-Z and a-z.

HuKeping commented on upstream.yaml:
```diff
@@ -74,10 +130,22 @@ autofdo:
   SCM: git
 
+autofdo_test:
+  upstream: https://github.com/google/autofdo.git
```
CI failed because there are two autofdo.git
```
fatal: destination path 'autofdo' already exists and is not an empty directory.
```

HuKeping reviewed (approved):
Finally, all green.
